### ISSUE TITLE
Mark a user review as "Helpful/Unhelpful"

### ISSUE THREAD

**Issue by pwalm** [mozilla/addons-frontend#5462]
Quick mock to spark discussion on the mark review as helpful/unhelpful feature:"
![add-ons - 2018-06-29 11 22 05](https://user-images.githubusercontent.com/6797635/42109111-66f0d2b6-7b91-11e8-8720-ff334798d145.jpg)

What it would look like on the extension detail page:
<img width="961" alt="screen shot 2018-06-29 at 11 42 26 am" src="https://user-images.githubusercontent.com/6797635/42109159-8f821d8e-7b91-11e8-99c7-61729a104103.png">

What it would look like on the "All Reviews" page:
<img width="944" alt="screen shot 2018-06-29 at 11 43 27 am" src="https://user-images.githubusercontent.com/6797635/42109197-ac0f1556-7b91-11e8-86ff-783d05ac3892.png">

**Comment by kumar303** [comment]
I like it. The issue is in the project now.

**Comment by willdurand** [comment]
@jvillalobos is it still something we want to do or can we close this issue?

**Comment by jvillalobos** [comment]
Yes, we still want this.

**Comment by willdurand** [comment]
> Yes, we still want this.

Ok cool, is this part of any roadmap/goal? Asking because it was part of Q4 2018 and we haven't made much changes here over the last 2 quarters.

**Comment by jvillalobos** [comment]
Yeah, it was part of that project but there wasn't enough time for it and the plans changed. It's not part of the current roadmap and I doubt we'll get to it this year.
